src/Interpreters/ClientInfo.cpp

```diff
@@ -60,6 +60,8 @@ void ClientInfo::write(WriteBuffer & out, const UInt64 server_protocol_revision)
     if (server_protocol_revision >= DBMS_MIN_REVISION_WITH_QUOTA_KEY_IN_CLIENT_INFO)
         writeBinary(quota_key, out);
 
+    writeBinary(distributed_depth, out);
```

Comment:
Looks like you forget about making it compatible with older versions, i.e.:
- increment VERSION_REVISION
- introduce DBMS_MIN_REVISION_WITH_DISTRIBUTED_DEPTH, set it to VERSION_REVISION

Reply:
Not `VERSION_REVISION`, it identifies a particular release and is incremented automatically when a new release is published.
The native protocol is versioned by the `DBMS_TCP_PROTOCOL_VERSION`.